operator-common/src/main/java/io/strimzi/operator/common/http/Liveness.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.common.http;
+
+/**
+ * A liveness check implemented by an application (e.g. an operator) and called by the
+ * {@link HealthCheckAndMetricsServer} when handling a health check request.
+ */
+public interface Liveness {
+
+    /**
+     * Indicates whether the controller is alive or not.
```

Comment:
isn't "the application" better? You are being generic later about it and then adding an example usage with a controller, or?